$OpenBSD: patch-kmymoney_plugins_kbanking_CMakeLists_txt,v 1.1 2017/01/20 09:14:12 zhuk Exp $
Unbreak KBanking plugin (at runtime, since we don't use --no-undefined for ld).
--- kmymoney/plugins/kbanking/CMakeLists.txt.orig	Thu Jan 19 15:52:34 2017
+++ kmymoney/plugins/kbanking/CMakeLists.txt	Thu Jan 19 16:02:29 2017
@@ -6,10 +6,9 @@ include_directories(${CMAKE_CURRENT_SOURCE_DIR}
                     ${CMAKE_CURRENT_BINARY_DIR}/dialogs
                     ${CMAKE_CURRENT_SOURCE_DIR}/widgets
                     ${CMAKE_CURRENT_BINARY_DIR}/widgets
-                    ${GWENHYWFAR_INCLUDE_DIRS}
                     ${AQBANKING_INCLUDE_DIRS}
-                    ${gwengui-cpp_INCLUDE_DIRS}
-                    ${gwengui-qt4_INCLUDE_DIRS}
+                    ${GWENGUI_QT4_INCLUDE_DIRS}
+                    ${GWENHYWFAR_INCLUDE_DIRS}
 )
 
 add_subdirectory( dialogs )
@@ -53,13 +52,12 @@ target_link_libraries(kmm_kbanking
                       kmm_plugin
                       payeeidentifier_iban_bic
                       payeeidentifier_nationalAccount
+                      ${LIBALKIMIA_LIBRARIES}
                       ${KDE4_KDEUI_LIBS}
                       ${QT_QTDECLARATIVE_LIBRARY}
-                      ${GWENHYWFAR_LIBRARIES}
-                      ${gwengui-qt4_LIBRARIES}
-                      ${gwengui-cpp_LIBRARIES}
                       ${AQBANKING_LIBRARIES}
-                      ${LIBALKIMIA_LIBRARIES}
+                      ${GWENHYWFAR_LIBRARIES}
+                      ${GWENGUI_QT4_LIBRARIES}
 )
 
 
